src/DynamoCoreWpf/Views/Core/WorkspaceView.xaml.cs

```diff
@@ -704,6 +714,15 @@ private HitTestResultBehavior DirectHitTestCallback(HitTestResult result)
             return HitTestResultBehavior.Continue;
         }
 
+        private void ShowInCanvasControl()
+        {
+            // Show InCanvas search just in case, when mouse is over workspace.
+            if (!this.IsMouseOver)
+                return;
+
+            InCanvasSearchBar.IsOpen = true;
+        }
+        
```

Comment:
As part of the consistency improvement, you may want to use a lambda for `Show` because `Close` is a lambda expression.